### DISCUSSION TITLE
2023-12-17-jp

### DISCUSSION THREAD

**Discussion by giscus** [tw93/tw93.github.io#161] [General]
# 日本东京&大阪游记 - Tw93

Hi，我是 Tw93，是一个前端工程师，会一点 Swift / Rust / Python，喜欢折腾各种新技术。Share11月底去日本玩了5天，第二次去日本玩，想用这篇文章来记录一下看到、听到、和本地朋友聊天等方式下和自己固有想法不一样的地方，或者说是我自己的独特感受，可能有偏差欢迎指出，也希望给后面去玩的朋友一些输入。

https://tw93.fun/2023-12-17/jp.html

**Comment by swizardlv** [comment]
看到这一段”问了问普遍认为的阶级跃迁是年薪超过 1000w 日元，大概人民币 50w 以上，"直觉是错了吧还真去查了一下汇率。十五年前在日本出差，有一个比我高很多级的同事（老板级别）每天出租车上下班。日本同事都艳羡不以，说是年收入1000万日元就可以.... 原来以为遥不可及.... 现在看中国稍好的程序员就可以达到了

**Reply by tw93** [reply]
社会福利好

**Comment by yuexueyang** [comment]
甚至是工地出来的车的车轮都是黑的

**Reply by tw93** [reply]
很神奇的一个地方了

**Comment by caijinyc** [comment]
感谢分享，以后去日本玩有的参考了！

**Reply by tw93** [reply]
谢谢喜欢，日本我感觉我会去第三次
